### PR TITLE
Fix Formatting and Update Intro

### PR DESCRIPTION
**Pull request by junweimoo**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -10,11 +10,18 @@ title: User Guide
...
+*MyInsuRec* is **the desktop app for financial advisors**. If you are a financial advisor looking for an app to better 
+manage your clients, meetings and products details, then look no further! *MyInsuRec* can boost your productivity 
+with features to quickly look up client and meetings details. Beyond that, *MyInsuRec* also has features to help you 
+improve your relationships with clients.
 
-*MyInsuRec* is optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, *MyInsuRec* can get your client, meeting and product management tasks done faster than traditional GUI apps.
+*MyInsuRec* is optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User 
+Interface (GUI). If you can type fast, *MyInsuRec* can get your client, meeting and product management tasks done 
+faster than traditional GUI apps!
```
Comment (ThomasHoooo) [inline docs/UserGuide.md:20]: This should be one line

File: docs/UserGuide.md [hunk 2]
```diff
@@ -10,11 +10,18 @@ title: User Guide
...
 
-*MyInsuRec* is available for the Windows, macOS and Linux operating systems.
+*MyInsuRec* is available for the Windows, macOS and Linux operating systems. To get started, simply head over to the 
+[installation guide](#3-installation-guide). This user guide can also serve as a reference for experienced 
+users, with a convenient [command summary](#7-command-summary).
```
Comment (ThomasHoooo) [inline docs/UserGuide.md:24]: same issue

File: docs/UserGuide.md [hunk 3]
```diff
@@ -10,11 +10,18 @@ title: User Guide
 
 ## 1. Introduction
 
-*MyInsuRec* is **the desktop app for financial advisors**. If you are a financial advisor looking for an app to better manage your clients, meetings and products details, then look no further. *MyInsuRec* can also boost your productivity with features to quickly look up client and meetings details. Beyond that, *MyInsuRec* also has features to help you improve your customer relations.
+*MyInsuRec* is **the desktop app for financial advisors**. If you are a financial advisor looking for an app to better 
+manage your clients, meetings and products details, then look no further! *MyInsuRec* can boost your productivity 
+with features to quickly look up client and meetings details. Beyond that, *MyInsuRec* also has features to help you 
+improve your relationships with clients.
```
Comment (ThomasHoooo) [inline docs/UserGuide.md:16]: this should be one line

**Review by ThomasHoooo — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 4]
```diff
@@ -10,11 +10,11 @@ title: User Guide
...
+*MyInsuRec* is **the desktop app for financial advisors**. If you are a financial advisor looking for an app to better manage your clients, meetings and products details, then look no further! *MyInsuRec* can boost your productivity with features to quickly look up client and meetings details. Beyond that, *MyInsuRec* also has features to help you improve your relationships with clients.
 
-*MyInsuRec* is optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, *MyInsuRec* can get your client, meeting and product management tasks done faster than traditional GUI apps.
+*MyInsuRec* is optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, *MyInsuRec* can get your client, meeting and product management tasks done faster than traditional GUI apps!
```
Comment (sikai00) [inline docs/UserGuide.md:15]: i will update to no CLI and GUI

File: docs/UserGuide.md [hunk 5]
```diff
@@ -501,6 +504,10 @@ MyInsuRec data are saved as a JSON file `[JAR file location]/data/myinsurec.json
 If your changes to the data file makes its format invalid, MyInsuRec will discard all data and start with an empty data file at the next run.
 </div>
 
+#### 5.5.3 Change the color scheme
+
+MyInsuRec comes with a light and dark color scheme. To change between the two, simple click on the View menu button and select the desired mode!
```
Comment (sikai00) [inline docs/UserGuide.md:509]: Add images later

**Pull request merged**